eos-gpt2mbr

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash -e
+# Copyright (C) 2016 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+get_last_char() {
```

Comment:
Now that this code is running under bash we can use bash's built in functionality to read the last char of a variable